ztp/source-crs/ClusterVersionUpdateGraph.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterVersion
+metadata:
+  name: version
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "90"
+spec:
+  channel: $channel
```

Comment:
If we are going to use one source-cr, then we should put an explicit value here (stable-4.10?) to help with image precaching. If we do so, then part of this bz would be resolved: https://bugzilla.redhat.com/show_bug.cgi?id=2058414#c2